Azure/azure-cli
# [RDBMS] Add flexible-server connect command

**Description**  
Adding flexible-server connect command.  This allows customer to validate a connection to the database and optionally run a query against the database using the passed in username and password to connect.  If database name is not provided the default db is used.  Server name is pulled from local-context if applicable.

**Testing Guide**  

1. Create a PG and/or MySQL server with full firewall access to client IP 
2. Run the command below to connect:

az postgres flexible-server connect -n moljain-testserver -u mjuser -p Experiment123 -c "select * from pg_user;" --output table
az mysql flexible-server connect -n moljain-testserver2 -u mjuser -p Experiment123 -c "select host, user from mysql.user;" --output table

This checklist is used to make sure that common guidelines for a pull request are followed.
